readme.md

```diff
@@ -137,26 +181,43 @@ $ npm -v
 
 * **Your Answer:**
 
----
+Hello, Node!
+Toan Bui
+C:\source\Summer2019\w1-node-ecosystem
```

Comment:
This doesn't describe what `path` is. `path` is a module built-in to Node that we can use to access information about the file system.